code/modules/goals/station_goals/bluespace_tap.dm

```diff
@@ -175,6 +173,22 @@
 		/obj/item/pizzabox,
 	)
 
+/obj/effect/spawner/lootdrop/bluespace_tap/mats
+	name = "materials"
+	lootcount = 70 //average miners should get the total of this
+	loot = list(
+		/obj/item/stack/ore/iron = 30,
+		/obj/item/stack/ore/glass = 30,
+		/obj/item/stack/ore/titanium = 15,
+		/obj/item/stack/ore/uranium = 15,
+		/obj/item/stack/ore/diamond = 13,
+		/obj/item/stack/ore/bluespace_crystal = 12,
+		/obj/item/stack/ore/bananium = 1, //as rare as possible
+		/obj/item/stack/ore/plasma = 18,
+		/obj/item/stack/ore/gold = 15,
+		/obj/item/stack/ore/silver = 15
+	)
```

Comment:
so i'm gonna be real with you, i don't think you understand how weights work, so just revert them and remove bananium entirely. if it's not naturally spawning on lavaland then theres no reason for it to come from this lootdrop